index.js

```diff
@@ -11,6 +11,7 @@ const HistoricAlert = require('./lib/historicAlert')
 const Location = require('./lib/location')
```

Comment:
Just a reminder to update the `CHANGELOG`.

Reply:
Gotcha!